skypy/galaxy/tests/test_luminosity.py

```diff
@@ -36,12 +36,10 @@ def calc_pdf(luminosity, z, alpha, a_m, b_m, absolute_magnitude_max=-16.0,
         luminosity_max = astro.luminosity_from_absolute_magnitude(
             absolute_magnitude_min)
         lum_star = lum._calculate_luminosity_star(z, a_m, b_m)
-        c = special.upper_incomplete_gamma(alpha + 1,
-                                           luminosity_min / lum_star)
-        d = special.upper_incomplete_gamma(alpha + 1,
-                                           luminosity_max / lum_star)
-        return 1. / lum_star * np.power(luminosity / lum_star, alpha) * np.exp(
-            - luminosity / lum_star) / (c - d)
+        c = special.gammaincc(alpha+1, luminosity_min/lum_star)
+        d = special.gammaincc(alpha+1, luminosity_max/lum_star)
+        return 1./lum_star*np.power(luminosity/lum_star, alpha)*np.exp(
+                -luminosity/lum_star)/(c - d)
```

Comment:
Here, (c-d) should also be multiplied with `Gamma(alpha+1)` using the regularised upper inc. gamma function, shouldn't, it?

Reply:
for the real PDF, yes, but the CDF is normalised so that is unnecessary

Reply:
True, but might be confusing in the future?